### ISSUE TITLE
Unable to get ServerOffersFolder!

### ISSUE THREAD

**Issue by titan-3217** [BobDaRoss/QuantumHangarIssues#21]
[QuantumHangar.log](https://github.com/BobDaRoss/QuantumHangarIssues/files/4668169/QuantumHangar.log)

Occurs if I click Browse Folder in HnagrMarket/Server offers tab.
I've created the directory after start torch.

**Comment by BobDaRoss** [comment]
I cannot replicate this issue. The folder is created on server startup. Even when I delete the folder and relaunch torch it creates the folder. Browse Folder always works for me.

I can think of a couple of possible issues none of which are certain:
1. You could be running an older version?
2. Do you have folder admin rights?

**Comment by titan-3217** [comment]
Steps that I did;
Started Torch with auto start disabled, I've installed the plugin via Torch download.
Then I started to configure QuantumHangar from UI when I realize that it requires a folder for market blueprints, so I create it under Instance directory as default.
Then I proceed to Hangar Market tab, and under Server offers tab I clicked Browser Folder. 

I was executing last version of Torch with admin rights.
Anyway is a minor issue, It worked properly after the error occurred and Torch was restarted.
